uikit/buttonV2/Button.tsx

```diff
@@ -0,0 +1,53 @@
+import { FunctionComponent, ButtonHTMLAttributes } from "react";
+import ButtonPrimary from "./ButtonPrimary";
+
+type ButtonPrimaryProps = {
+    size?: "md" | "lg" | "xl";
+    type?: "fab" | "default";
+} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, "size" | "type">;
```

Comment:
I think we are missing the `variant` props in here

Reply:
ups